examples/simple_eth.c

```diff
@@ -0,0 +1,75 @@
+#include <ingenialink/ingenialink.h>
+#include <ingenialink/registers.h>
+#include <time.h>
+#include <stdio.h>
+
+
+int main(int argc, const char *argv[])
+{
+	il_net_t *net = NULL;
+	il_servo_t *servo = NULL;
+	char* dict_path = "XXX";
+	char* target_ip = "192.168.2.22";
+	int port = 1061;
```

Comment:
Follow [style guide ](https://github.com/ingeniamc/ingenia-style-guide/blob/master/c_cpp_style.md#types) for types.